lib/ttt/board.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Ttt.Board do
+  def create() do
+    ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
+  end
+
+  def create(board, space, marker) do
```

Comment:
Is this function going to be used for creating a new board, or only for putting markers? If the latter, then not sure about the naming. Also, good job using Elixir's ability to define different clauses for `create` in your first PR!